app/assets/stylesheets/table/_body.scss

```diff
@@ -0,0 +1,14 @@
+.table > tbody > tr > td {
```

Comment:
Minor changes needed here on all `/table/**.scss` files.

Think of the expenses table view as a single isolated component: Expenses Table component (or smt).
Even though we may have to reuse this table style across the app, for now we'll just focus on this component.

With that in mind, do this:
1. add a class like `.expenses-table` that wrap the whole table contents;
2. create a `/stylesheets/expenses/_table.scss` and add a reference to it;
3. move `_body.scss` and `_head.scss` content to the new scss file and update the `.table` refs to `.expenses-table`;
4. on top of the scss file add some variable names to hold the colors; so later we can extract to a global style file easier.